### DISCUSSION TITLE
Please make Firefox Mask for Chromium

### DISCUSSION THREAD

**Discussion by gunir** [denschub/chrome-mask#21] [Ideas]
It would be great for people who use Chromium but want to say they support Firefox, webmasters must support Firefox, it works in a different way than Chrome Mask:

- It's on by default on all page, and can be disabled per site

Thanks for listening, there's a lot of Chromium users who still want to support Firefox, most website owners don't even know that the user changed UA, only the top of the foodchain like Cloudflare, Akamai can.

This is my idea of the extension icon:
![image](https://github.com/user-attachments/assets/9248ef4a-fa96-4f9a-a9f6-2c3bbb668197)

**Comment by skyschub** [comment]
I understand the request, and I appreciate the intentions, but no. The only reasonable way to support Firefox is to use Firefox.

There are many ways that sites and analytics gatherers can identify the browsers in way that no addon could ever spoof. For example, a site could check if the browser supports WebUSB - and if it does, it has to be something else than Firefox. There's also a lot of other mechanisms.

Blanket spoofing the UA in Chrome doesn't really help Firefox in any meaningful way. But it will make user's experience a lot worse. Some things will randomly break, users will see a ton of extra CAPTCHAs because their browsers are sending "mixed signals", and a lot more.

If someone wants to build an extension like this, cool, but I'm not going to do it.
